modules/custom/az_carousel/config/install/core.entity_view_display.node.az_carousel_item.default.yml

```diff
@@ -17,84 +17,86 @@ third_party_settings:
       children:
         - field_az_media_image
         - group_slide_details
+      label: 'Carousel Slide'
       parent_name: ''
+      region: content
       weight: 1
       format_type: html_element
-      region: content
       format_settings:
+        classes: 'az-carousel-slide d-flex flex-column justify-content-start'
+        id: ''
         element: div
         show_label: false
         label_element: h3
         label_element_classes: ''
         attributes: ''
         effect: none
         speed: fast
-        id: ''
-        classes: 'az-carousel-slide d-flex flex-column justify-content-start'
-      label: 'Carousel Slide'
     group_slide_details:
       children:
         - smart_title
         - group_text_size_h4
         - field_az_link
+      label: 'Slide Details'
       parent_name: group_carousel_item
+      region: content
       weight: 3
       format_type: html_element
-      region: content
       format_settings:
+        classes: 'az-carousel-slide-details px-4 py-3 d-flex flex-column justify-content-center'
+        id: ''
         element: div
         show_label: false
         label_element: h3
         label_element_classes: ''
         attributes: ''
         effect: none
         speed: fast
-        id: ''
-        classes: 'az-carousel-slide-details px-4 py-3 d-flex flex-column justify-content-center'
-      label: 'Slide Details'
     group_text_size_h4:
       children:
         - field_az_carousel_short_summary
+      label: 'Text Size H4'
       parent_name: group_slide_details
+      region: content
       weight: 2
       format_type: html_element
-      region: content
       format_settings:
+        classes: 'h4 mt-0 mb-3 text-white'
+        show_empty_fields: false
+        id: ''
         element: div
         show_label: false
         label_element: h3
         label_element_classes: ''
         attributes: ''
         effect: none
         speed: fast
-        id: ''
-        classes: 'h4 mt-2 mb-3 text-white'
-      label: 'Text Size H4'
   smart_title:
     enabled: true
     settings:
+      smart_title__link: false
       smart_title__tag: h2
       smart_title__classes:
         - h1
-        - my-0
+        - mt-0
+        - mb-2
```

Comment:
Maybe we could keep the title margin as `my-0` and then add a top margin to each of the other two elements (the summary and call-to-action)? That could prevent any visual change if the slide only has a title. This might require creating another div for the call to action.

Reply:
Yes, agreed. I have updated my implementation with these changes.